### DISCUSSION TITLE
How safe is it to store db.sqlite3 on a third party service?

### DISCUSSION THREAD

**Discussion by dick-jo** [dani-garcia/vaultwarden#1798] [Q&A]
Currently running vaultwarden as a Docker container on a Synology 920+.

If I set up a basic cloud backup job to sync my vaultwarden data to a third party (dropbox) service - how safe is this?

I've tested restoring historic db.sqlite3 files as a proof of concept for this backup method and seems to work fine, but this is new technology for me so I was wondering what degree of liability potentially exposing these files runs. If any.

**Comment by BlackDex** [comment]
Almost everything in the database is encrypted. So your passwords, usernames etc... are safe.

Though, i personally would still encrypt backups stored on an external provider just to be safe. Not sure what the best options are for you, because it all depends on from where to where you store it, what you are comfortable with. Best to search regarding that on the internet.

**Reply by MilesTEG1** [reply]
> Almost everything in the database is encrypted. So your passwords, usernames etc... are safe.
> 
> Though, i personally would still encrypt backups stored on an external provider just to be safe. Not sure what the best options are for you, because it all depends on from where to where you store it, what you are comfortable with. Best to search regarding that on the internet.

I join @BlackDex on the backup encryption.
I have the same setup as you @dick-jo : Vaultwarden installed in a docker container on a synology DS920+
Every night I do a backup with Hyperbackup with encryption on a second volume, and on a google drive account with encryption too.
I encrypt every backup I do on google Drive. You should do the same 😃
